src/SWP/Bundle/MultiTenancyBundle/Resources/config/services.yml

```diff
@@ -17,7 +17,6 @@ services:
     swp_multi_tenancy.tenant_resolver:
         class: '%swp_multi_tenancy.tenant_resolver.class%'
         arguments:
-            - '%domain%'
```

Comment:
it is used for migration (if there is no domain then it's filled with this). We can depreciate it but should stay until next minor.